packages/runtime/src/core.ts

```diff
@@ -107,6 +125,8 @@ export class FederationHost {
         {
           id: string;
           error: unknown;
+          from: 'build' | 'runtime';
```

Comment:
Why do I need to add a from

Reply:
loadRemote will return factory in build and return module in runtime . 

So when uses want to set errorLoadRemote hook , it needs to know where the api(loadRemote) is called